WORKSPACE

```diff
@@ -18,88 +32,53 @@ bind(
 
 load("@rules_scala_annex//rules/scala:workspace.bzl", "scala_register_toolchains", "scala_repositories")
 scala_repositories()
+load("@annex//:defs.bzl", annex_pinned_maven_install = "pinned_maven_install")
+annex_pinned_maven_install()
 scala_register_toolchains()
 
-skylib_version = "8cecf885c8bf4c51e82fd6b50b9dd68d2c98f757"  # update this as needed
-http_archive(
-    name = "bazel_skylib",
-    sha256 = "d54e5372d784ceb365f7d38c3dad7773f73b3b8ebc8fb90d58435a92b6a20256",
-    strip_prefix = "bazel-skylib-{}".format(skylib_version),
-    type = "zip",
-    url = "https://github.com/bazelbuild/bazel-skylib/archive/{}.zip".format(skylib_version),
+# JDK stuff needed for Scala 2.11
```

Comment:
Might want to expand on this comment a bit. Java 8 needed because Scala 2.11 doesn't support Java 11 well.

Reply:
I'll update the comment